### PR TITLE
iproute2: update to 6.1.0

### PR DESCRIPTION
**Pull request by PolynomialDivision**

### CODE REVIEW

File: package/network/utils/iproute2/Makefile [hunk 1]
```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=iproute2
-PKG_VERSION:=6.0.0
+PKG_VERSION:=v6.1.0
```
Comment (robimarko) [inline package/network/utils/iproute2/Makefile:11]: Will this break OPKG?

Maybe keep the version without v and just add the v in `PKG_SOURCE`

Comment (PolynomialDivision) [reply package/network/utils/iproute2/Makefile]: Changed that

File: package/network/utils/iproute2/Makefile [hunk 2]
```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=iproute2
-PKG_VERSION:=6.0.0
+PKG_VERSION:=6.1.0
 PKG_RELEASE:=$(AUTORELEASE)
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
```
Comment (hauke) [inline package/network/utils/iproute2/Makefile:14]:
```suggestion
PKG_SOURCE:=$(PKG_NAME)-v$(PKG_VERSION).tar.xz
```
This fixes the download of the package.

File: package/network/utils/iproute2/Makefile [hunk 3]
```diff
@@ -8,12 +8,12 @@
...
 PKG_SOURCE_URL:=@KERNEL/linux/utils/net/iproute2
-PKG_HASH:=523139e9e72aec996374fa2de74be4c53d2dd05589488934d21ff97bae19580a
+PKG_HASH:=b58b95f34c5b6e8171d1833ccfcf43c86aaa064c99419bd1b2a2eb7ee741f089
```
Comment (hauke) [inline package/network/utils/iproute2/Makefile:16]: The package hash is wrong, the build fails like this:
````
+ curl -f --connect-timeout 20 --retry 5 --location https://cdn.kernel.org/pub/linux/utils/net/iproute2/iproute2-6.1.0.tar.xz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  884k  100  884k    0     0  31.9M      0 --:--:-- --:--:-- --:--:-- 33.2M
Hash of the downloaded file does not match (file: 5ce12a0fec6b212725ef218735941b2dab76244db7e72646a76021b0537b43ab, requested: b58b95f34c5b6e8171d1833ccfcf43c86aaa064c99419bd1b2a2eb7ee741f089) - deleting download.
+ curl -f --connect-timeout 20 --retry 5 --location https://download.xs4all.nl/ftp.kernel.org/pub/linux/utils/net/iproute2/iproute2-6.1.0.tar.xz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
````
```suggestion
PKG_HASH:=5ce12a0fec6b212725ef218735941b2dab76244db7e72646a76021b0537b43ab
```

**Pull request merged**